pip/wheel.py

```diff
@@ -668,19 +668,24 @@ def _build_one(self, req, output_dir):
                     logger.info('Stored in directory: %s', output_dir)
                     return wheel_path
                 except:
-                    return None
+                    pass
```

Comment:
Correction - if the wheel build fails, that results in ```__build_one``` returning ```False```. The ```except``` clause here is solely if the wheel build *succeeded*, but the process of moving the wheel into ```wheel_path``` (the cache, presumably?) failed.

In that case, agreed a debug of the exception might be useful. But logging at warning level that caching the wheel failed is *also* probably useful. Both points are independent of this patch, though, so I think they should be dealt with separately.